OSDOCS-3014: Add links to pruning for disconnected OLM (#39340)
openshift/openshift-docs: installing/installing-mirroring-installation-images.adoc

* xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]

[id="olm-mirror-catalog-prerequisites_installing-mirroring-installation-images"]
=== Prerequisites

> Mirroring Operator catalogs for use with disconnected clusters has the following prerequisites:

Reviewer: ```suggestion
Using mirroring Operator catalogs with disconnected clusters has the following prerequisites:
```

Just a small suggestion that may make it a little easier to read. Please ignore, if it changes the meaning of the sentence though.

Author: Yeah, changes it a bit.

* Workstation with unrestricted network access.
* `podman` version 1.9.3 or later.
* If you want to filter, or _prune_, the default catalog and selectively mirror only a subset of Operators, see the following sections:
** xref:../cli_reference/opm/cli-opm-install.adoc#cli-opm-install[Installing the opm CLI]
** xref:../operators/admin/olm-restricted-networks.adoc#olm-pruning-index-image_olm-restricted-networks[Filtering a SQLite-based index image]
ifndef::openshift-origin[]
* If you want to mirror a Red Hat-provided catalog, run the following command on your workstation with unrestricted network access to authenticate with `registry.redhat.io`:
+
[source,terminal]
----
$ podman login registry.redhat.io
----
endif::[]

> * Access to mirror registry that supports

Reviewer: ```suggestion
* Access to a mirror registry that supports
```

link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2].

> * Decide which namespace on your mirror registry you will use to store the mirrored Operator content. For example, you might create an `olm-mirror` namespace.

Reviewer: ```suggestion
* Decide which namespace to use, on your mirror registry, to store the mirrored Operator content. For example, you might create an `olm-mirror` namespace.
```

Author: Changed to:

> On your mirror registry, decide which namespace to use for storing mirrored Operator content.

Author: Changed to:

> On your mirror registry, decide which namespace to use for storing mirrored Operator content.
